tasks/nms/install-debian.yml

```diff
@@ -27,10 +27,28 @@
   register: nms_install_state
   when: nms_cleanup_status is not defined
 
-- name: (Debian/Ubuntu) Make sure NGINX Instance Manager is running
+- name: (Debian/Ubuntu) NMS Modules
+  ansible.builtin.apt:
+    name: "{{ item.name }}={{ '*' if (item.version is not defined or item.version == '') else item.version }}"
```

Comment:
You won't be able to upgrade an existing installation with the `=*` at the end of the package name. I would leave off `=*` if we are installing/upgrading the latest. I confirmed the issue on a RHEL OS. I don't remember if this was true for Debian or not but might be good to keep this structure consistent between the two OS Families.

I tested this by Installing NMS version 2.6.0. Then ran the following task:
```yaml
  tasks:
    - name: (Amazon Linux/CentOS/Oracle Linux/RHEL) Upgrade NGINX Instance Manager
      ansible.builtin.yum:
        name: "nms-instance-manager-*"
        update_cache: true
        state: "latest"
        update_only: true
```

This resulted in 
```shell
$ ansible-playbook -i inventory nms-install.yml 

PLAY [Configure NGINX Management Suite] *************************************************************************************************************

TASK [Gathering Facts] ******************************************************************************************************************************
ok: [10.155.217.36]

TASK [(Amazon Linux/CentOS/Oracle Linux/RHEL) Upgrade NGINX Instance Manager] ***********************************************************************
ok: [10.155.217.36]

PLAY RECAP ******************************************************************************************************************************************
10.155.217.36              : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```

I updated the task by removing the `-*` part.
```yaml
  tasks:
    - name: (Amazon Linux/CentOS/Oracle Linux/RHEL) Upgrade NGINX Instance Manager
      ansible.builtin.yum:
        name: "nms-instance-manager"
        update_cache: true
        state: "latest"
        update_only: true
```

I was able to upgrade after making this change
```shell
$ ansible-playbook -i inventory nms-adm-install.yml 

PLAY [Configure NGINX Management Suite] *************************************************************************************************************

TASK [Gathering Facts] ******************************************************************************************************************************
ok: [10.155.217.36]

TASK [(Amazon Linux/CentOS/Oracle Linux/RHEL) Upgrade NGINX Instance Manager] ***********************************************************************
changed: [10.155.217.36]

PLAY RECAP ******************************************************************************************************************************************
10.155.217.36              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

Reply:
Thanks @jswongF5, this has been addressed in the latest changes